assets/css/about.css

```diff
@@ -0,0 +1,389 @@
+  font-weight: 600;
+}
+
+.past .red-line {
+  width: 50px;
+  border-color: #ec5242;
+}
+
+.past .list {
+  list-style: none;
+  display: flex;
+  flex-wrap: wrap;
+  margin: 40px -24px;
+  padding: 0;
+}
+
+.past .list li .card {
+  margin: 24px;
+  position: relative;
+}
+
+.past .list li .card img {
+  width: 100%;
+  height: 100%;
+}
+
+.past .list li .card .detail {
+  position: absolute;
+  top: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
+  color: #fff;
+  background-color: #ec5342cf;
+}
+
+.past .list li .card .detail h3 {
+  font-size: 25px;
+}
+
+.past .list li .card .detail p {
+  font-size: 20px;
+  margin-top: 5px;
+}
+
+.partners {
+  background: #272a31;
+  color: #fff;
+  padding: 36px 10px;
+}
+
+.partners .title-div {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.partners .title {
+  font-size: 24px;
+  font-weight: 600;
+}
+
+.partners .red-line {
+  width: 50px;
+  border-color: #ec5242;
+}
+
+.partners .list {
+  list-style: none;
+  padding: 0;
+  margin: 24px 0;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+}
+
+.partners .list li svg {
+  margin: 10px 16px;
+  width: 120px;
+}
+
+.footer-bg {
+  background: #272a31;
+  color: #fff;
+}
+
+.footer {
+  display: flex;
+  padding: 32px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo-section {
+  border-bottom: 1px solid #dbdde1;
+  padding: 48px 0;
+}
+
+.logo {
+  text-decoration: none;
+  font-size: 18px;
+  font-family: Cocogoose, "Courier New", Courier, monospace;
+  color: #272a31;
+}
+
+.des {
+  margin: 24px 0;
+  font-size: 16px;
+  line-height: 25px;
+  text-align: center;
+  padding: 0 24px;
+}
+
+.logo-des {
+  border: 1px solid #5f6167;
+  padding: 34px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 80%;
+  margin: 32px auto;
+}
+
+.logo-des .logo {
+  font-size: 45px;
+}
+
+.logo-des .time {
+  border-bottom: 1px solid #5f6167;
+  border-top: 1px solid #5f6167;
+  padding: 10px 0;
+  display: inline-block;
+}
+
+.footer .logo {
+  color: #fff;
+}
+
+.mobile-menu {
+  background: #ec5342c7;
+  z-index: 50;
+  height: 100%;
+  width: 100%;
+  position: fixed;
+  backdrop-filter: blur(2px);
+  display: none;
+  top: 0;
+}
+
+.mobile-menu > .close-modal {
+  background: transparent;
+  border: none;
+  color: white;
+  position: absolute;
+  right: 27px;
+  top: 8px;
+  font-size: 40px;
+}
+
+.mobile-menu > nav {
+  padding: 86px 98px 86px 28px;
+}
+
+.mobile-menu > nav > ul {
+  list-style: none;
+}
+
+.mobile-menu > nav > ul > li {
+  margin-bottom: 40px;
+  text-align: left;
+}
+
+.mobile-menu > nav > ul > li > a {
+  color: white;
+  font-size: 32px;
+  line-height: 44px;
+  text-decoration: none;
+}
+
+.desktop-nav {
+  display: none;
+}
+
+@media (min-width: 768px) {
+  .container {
+    width: 1024px;
```

Comment:
- [ ] OPTIONAL: I strongly recommend to delete this fixed width, because it is breaking the page in smaller screens than `1024px`: 

https://user-images.githubusercontent.com/77038610/163546868-fae00659-b322-4eb4-938a-6f54cba7b58d.mp4

Reply:
Thanks 😊 🙏